contracts/LosslessReporting.sol

```diff
@@ -208,7 +211,7 @@ contract LosslessReporting is Initializable, ContextUpgradeable, PausableUpgrade
     /// @param account Potential malicious address
     function secondReport(uint256 reportId, address account) public notBlacklisted whenNotPaused {
         require(account != address(0), "LSS: Cannot report zero address");
-        require(!losslessGovernance.isReportSolved(reportId), "LSS: Report already solved.");
+        require(!losslessGovernance.isReportSolved(reportId) && !losslessGovernance.reportResolution(reportId), "LSS: Report already solved.");
```

Comment:
Did this not require any updates in the tests?

Reply:
Cannot we have the same validation statement as in the _report_ function ? 

`require(reportId == 0 || 
                reportTimestamps[reportId] + reportLifetime < block.timestamp || 
                losslessGovernance.isReportSolved(reportId) && 
                !losslessGovernance.reportResolution(reportId), "LSS: Report already exists");
`
?